src/app/app.module.ts

```diff
@@ -18,6 +19,7 @@ import { GroceryModule } from 'src/app/grocery/grocery.module';
   ],
   declarations: [
     AppComponent,
+    NavigationComponent,
```

Comment:
It should be better to create core module or app layout module, or smth like this and move navigation component there

Reply:
Not quite, navigation isn't a part of the core or layout module. It's a part of the whole application and it's better to keep it in the application module.